masterlist.yaml

```
@@ -20453,16 +20453,29 @@ plugins:
     url:
       - link: 'https://www.nexusmods.com/oblivion/mods/33698/'
         name: 'DLC Frostcrag Cobl Addons'
+    msg:
+      - <<: *alreadyInX
+        subs: [ '[Frostcrag Spire Coblized](https://www.nexusmods.com/oblivion/mods/52608/)' ]
+        condition: 'active("DLCFrostcragCoblAlchemyTable.esp")'
```

```suggestion
        subs: [ '**DLCFrostcragCoblAlchemyTable.esp**' ]
        condition: 'active("DLCFrostcragCoblAlchemyTable.esp") and version("DLCFrostcragCoblAlchemyTable.esp", "2.0", >=)'
```

If they already have the plugin, you don't need to link to it. It's also clearer what plugin contains this plugin if you reference it by filename. You could also add the version check here to avoid confusion with users of older versions.